### DISCUSSION TITLE
What "identifier" should I pick?

### DISCUSSION THREAD

**Discussion by carcinocron** [tauri-apps/tauri#5976] [Q&A]
Are there namespace rules for this kind of thing? Best practices?

**Comment by FabianLars** [comment]
In general it follows the reverse domain name notation. We don't recommend it, but it _should_ be possible to use other formats (or just the app name) in case you don't care about app store support (they generally require this format iirc), but it's not officially supported by us...
The most important thing is that it _must_ be unique for each app.

**Comment by JonasKruckenberg** [comment]
This would be great to add to the docs FAQ as-is. How do i bookmark something 😂

**Reply by FabianLars** [reply]
I always leave the tab open in a separate window and then close that after a month of ignoring it. 10/10 would recommend.

**Comment by valentimarco** [comment]
In tauri 2.8.2 under linux, if you pick a identifier different from the `reverse domain` you get this error:
```
(nuxt-beats:57669): Gtk-CRITICAL **: 16:05:35.464: gtk_application_new: assertion 'application_id == NULL || g_application_id_is_valid (application_id)' failed

thread 'main' panicked at /home/marco/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/gtk-0.18.2/src/auto/application.rs:13:1:
assertion failed: !ptr.is_null()
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Here the conf used:
```json
{
  "$schema": "https://schema.tauri.app/config/2.0.0",
  "productName": "NuxtBeats",
  "identifier": "nuxtbeaters",
  "app": {
    "enableGTKAppId": true,
    "windows": [
      {
        "title": "NuxtBeats",
        "width": 1366,
        "height": 768,
        "minWidth": 375,
        "minHeight": 667,
        "resizable": true,
        "decorations": false,
        "fullscreen": false
      }
    ]
  },
  "bundle": {
    "resources": []
  },
  "plugins": {}
}
```

UPDATE: Set `enableGTKAppId` to false or not even declare it (by default is false) and the window class is set as the indentifier, use `xprop` or `wlprop` to double check it
